framework/src/wifi_manager/wifi_manager.c

```diff
@@ -28,14 +30,21 @@
 #include "wifi_utils.h"
 #include "wifi_profile.h"
 
-#define NUM_OF_STATE_HANDLER 8 // the number of states
+#undef ndbg
+#define ndbg printf
+#undef nvdbg
+#define nvdbg printf
+
+#define NUM_OF_STATE_HANDLER 11 // the number of states
```

Comment:
The number of states is 11.
but, the number of handlers is not 11, but 10.

Reply:
Ok I will fix